app/src/functions/simirality.ts

```diff
@@ -0,0 +1,19 @@
+export const postSimirality = async (assignment: string, words: string[]) => {
+    // TODO クライアントサイドかサーバーサイドかでURLを変更する
+    // サーバーサイドの場合は、http://FastAPI:9004/similarity
+    const response = await fetch('http://localhost:9004/similarity', {
+        method: 'POST',
+        headers: {
+        'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ asignmentWord: assignment, words: words }),
```

Comment:
```suggestion
        body: JSON.stringify({ assignmentWord: assignment, words: words }),
```
ここmain.pyも直す必要があるから根気があれば直してほしい。。。。